.github/workflows/build_pkg_runner.yml

```diff
@@ -0,0 +1,32 @@
+name: Package Built by devenv
+on:
+  schedule:
+    - cron: '21 18 * * 5'
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+        matrix:
+          os: [ubuntu-18.04, macos-latest]
```

Comment:
Why ubuntu 18.04 but not 20.04?

Reply:
Simply following the principle of "step by step". I am still working on the CI failure of SOLVCON build, which uses 18.04. By using 18.04 here for scotch build, I can monitor those suspicious packages which may raise issue during SOLVCON build.

We may wait for longer after the whole CI jobs are upgraded to use 20.04 and then update this CI job. Or we can land this job, and update this CI job to 20.04 when the whole CI jobs are ready to use 20.04. Either way, it's ok with me.

Reply:
fwiw, by using 20.04 we still fail the CI with git branch on macOS (and Ubuntu passes)  https://github.com/tai271828/devenv/actions/runs/1768260365

Reply:
Good point!